staging/copy.sh

```diff
@@ -186,6 +192,12 @@ find "${CLIENT_REPO_TEMP}" -type f \( \
 
 echo "remove cyclical godep"
 rm -rf "${CLIENT_REPO_TEMP}/_vendor/k8s.io/client-go"
+# If godep cannot find dependent packages in the regular GOPATH, it will search
+# in staging/. If successful, godep will copy the dependents to
```

Comment:
in the secondary GOPATH staging/. If successful, godep will wrongly copy the dependents relative to the primary GOPATH (looks like a godep bug), creating the `${CLIENT_REPO_TEMP}/staging` dir.